fc-community-contracts/src/FactchainNFT.sol

```diff
@@ -22,11 +23,12 @@ interface IFactchainSFTEvents {
 /// @author Pierre HAY
 /// @notice
 /// @dev
-contract FactchainNFT is OwnableUpgradeable, ERC721URIStorageUpgradeable, UUPSUpgradeable, IFactchainSFTEvents {
+contract FactchainNFT is OwnableUpgradeable, AccessControlUpgradeable, ERC721URIStorageUpgradeable, UUPSUpgradeable, IFactchainSFTEvents {
```

Comment:
not related but since it's probably our last Solidity PR before launching, can we rename factsLitography into `truthFragments` to reflect the manifesto naming

Reply:
sure!